lib/dogma/rules/line_length.ex

```diff
@@ -6,21 +6,23 @@ defmodule Dogma.Rules.LineLength do
   alias Dogma.Script
   alias Dogma.Error
 
-  def test(script) do
-    Enum.reduce( script.lines, script, &check_line(&1, &2) )
+  def test(script, max_length: max) do
+    Enum.reduce( script.lines, script, get_length_checker(max) )
   end
-
-  def too_long?(num) do
-    num > 80
+  
+  def test(script) do
+    Enum.reduce( script.lines, script, get_length_checker() )
   end
 
-  defp check_line({i, line}, script) do
-    len = String.length( line )
-    if too_long?( len ) do
-      err = error(len, i)
-      Script.register_error( script, err )
-    else
-      script
+  def get_length_checker(max \\ 80) do
```

Comment:
renamed because it's now a function that returns the check_line function. Think it needs a better name though.
Public by mistake. I'll update this as it doesn't need to be and shouldn't be public.